dev-infra/commit-message/config.ts

```diff
@@ -39,36 +39,66 @@ export enum ScopeRequirement {
 
 /** A commit type */
 export interface CommitType {
+  value: string;
+  name: string;
+  short: string;
   scope: ScopeRequirement;
 }
 
 /** The valid commit types for Angular commit messages. */
 export const COMMIT_TYPES: {[key: string]: CommitType} = {
   build: {
+    name: 'build - Changes to local repository build system and tooling',
+    value: 'build',
+    short: 'build',
     scope: ScopeRequirement.Forbidden,
   },
   ci: {
+    name: 'ci - Changes to CI configuration and CI specific tooling',
+    value: 'ci',
+    short: 'ci',
     scope: ScopeRequirement.Forbidden,
   },
   docs: {
+    name: 'docs - Changes which exclusively change documentation.',
+    value: 'docs',
+    short: 'docs',
     scope: ScopeRequirement.Optional,
   },
   feat: {
+    name: 'feat - Creates a new feature',
+    value: 'feat',
+    short: 'feat',
     scope: ScopeRequirement.Required,
   },
   fix: {
+    name: 'fix - Fixes a previously discovered failure/bug',
+    value: 'fix',
+    short: 'fix',
     scope: ScopeRequirement.Required,
   },
   perf: {
+    name: 'perf- Improves performance without any change in API or output',
+    value: 'perf',
+    short: 'perf',
     scope: ScopeRequirement.Required,
   },
   refactor: {
+    name: 'refactor - Refactor without any change in API or output (includes code style changes)',
+    value: 'refactor',
+    short: 'refactor',
     scope: ScopeRequirement.Required,
   },
   release: {
+    name: 'release - A release point in the repository',
+    value: 'release',
+    short: 'release',
     scope: ScopeRequirement.Forbidden,
   },
   test: {
+    name: 'test - Adds tests to better coverage missing test cases or improve testing method',
```

Comment:
Would this read better like the following? I added `corrections` as that matches with what we have in `CONTRIBUTIONS.md`

```suggestion
    name: 'test - Improvements or corrections made to the project's test suite',
```